debian/copyright

```diff
@@ -0,0 +1,26 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: childe
+Upstream-Contact: Bence Ferdinandy <bence@ferdinandy.com>, Regolith Linux <regolith.linux@gmail.com>
+Source: regolith-linux.org
+
+Files: *
```

Comment:
I might be misunderstanding something here, but I think these files will _not_ be shipped with this package? Just a child binary?

Reply:
Debian continues to surprise me but my understanding is that this field has no impact on what is packaged where.  More generally, the Debian build scripts produce a source and optionally a binary package (for the host architecture) from the package metadata.  Any number of binary packages can be defined, and the definition of the contents of each binary is stored in a file `<pkg root>/debian/<binary package name>.install` where the actual definition of the binary package comes from the `<pkg root>/debian/control` file.